2017-03-31 11:56:30 jdbc[4]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-193]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:184)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:126)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:99)
	at org.h2.Driver.connect(Driver.java:69)
	at org.apache.commons.dbcp.DriverConnectionFactory.createConnection(DriverConnectionFactory.java:38)
	at org.apache.commons.dbcp.PoolableConnectionFactory.makeObject(PoolableConnectionFactory.java:582)
	at org.apache.commons.dbcp.BasicDataSource.validateConnectionFactory(BasicDataSource.java:1556)
	at org.apache.commons.dbcp.BasicDataSource.createPoolableConnectionFactory(BasicDataSource.java:1545)
	at org.apache.commons.dbcp.BasicDataSource.createDataSource(BasicDataSource.java:1388)
	at org.apache.commons.dbcp.BasicDataSource.getConnection(BasicDataSource.java:1044)
	at org.apache.commons.dbutils.AbstractQueryRunner.prepareConnection(AbstractQueryRunner.java:175)
	at org.apache.commons.dbutils.QueryRunner.query(QueryRunner.java:286)
	at azkaban.executor.JdbcExecutorLoader.fetchExecutor(JdbcExecutorLoader.java:854)
	at azkaban.executor.ExecutorManager.setupExecutors(ExecutorManager.java:203)
	at azkaban.executor.ExecutorManager.<init>(ExecutorManager.java:126)
	at azkaban.webapp.AzkabanWebServer.loadExecutorManager(AzkabanWebServer.java:281)
	at azkaban.webapp.AzkabanWebServer.<init>(AzkabanWebServer.java:202)
	at azkaban.webapp.AzkabanWebServer.main(AzkabanWebServer.java:762)
	at azkaban.soloserver.AzkabanSingleServer.main(AzkabanSingleServer.java:53)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-193]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 22 more
2017-03-31 12:06:13 jdbc[6]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-193]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:184)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:126)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:99)
	at org.h2.Driver.connect(Driver.java:69)
	at org.apache.commons.dbcp.DriverConnectionFactory.createConnection(DriverConnectionFactory.java:38)
	at org.apache.commons.dbcp.PoolableConnectionFactory.makeObject(PoolableConnectionFactory.java:582)
	at org.apache.commons.dbcp.BasicDataSource.validateConnectionFactory(BasicDataSource.java:1556)
	at org.apache.commons.dbcp.BasicDataSource.createPoolableConnectionFactory(BasicDataSource.java:1545)
	at org.apache.commons.dbcp.BasicDataSource.createDataSource(BasicDataSource.java:1388)
	at org.apache.commons.dbcp.BasicDataSource.getConnection(BasicDataSource.java:1044)
	at azkaban.database.AbstractJdbcLoader.getDBConnection(AbstractJdbcLoader.java:69)
	at azkaban.project.JdbcProjectLoader.getConnection(JdbcProjectLoader.java:1548)
	at azkaban.project.JdbcProjectLoader.fetchAllActiveProjects(JdbcProjectLoader.java:75)
	at azkaban.project.ProjectManager.loadAllProjects(ProjectManager.java:93)
	at azkaban.project.ProjectManager.<init>(ProjectManager.java:86)
	at azkaban.webapp.AzkabanWebServer.loadProjectManager(AzkabanWebServer.java:275)
	at azkaban.webapp.AzkabanWebServer.<init>(AzkabanWebServer.java:203)
	at azkaban.webapp.AzkabanWebServer.main(AzkabanWebServer.java:762)
	at azkaban.soloserver.AzkabanSingleServer.main(AzkabanSingleServer.java:53)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:144)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-193]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 27 more
2017-03-31 12:09:36 jdbc[6]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-193]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:184)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:126)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:99)
	at org.h2.Driver.connect(Driver.java:69)
	at org.apache.commons.dbcp.DriverConnectionFactory.createConnection(DriverConnectionFactory.java:38)
	at org.apache.commons.dbcp.PoolableConnectionFactory.makeObject(PoolableConnectionFactory.java:582)
	at org.apache.commons.dbcp.BasicDataSource.validateConnectionFactory(BasicDataSource.java:1556)
	at org.apache.commons.dbcp.BasicDataSource.createPoolableConnectionFactory(BasicDataSource.java:1545)
	at org.apache.commons.dbcp.BasicDataSource.createDataSource(BasicDataSource.java:1388)
	at org.apache.commons.dbcp.BasicDataSource.getConnection(BasicDataSource.java:1044)
	at azkaban.database.AbstractJdbcLoader.getDBConnection(AbstractJdbcLoader.java:69)
	at azkaban.project.JdbcProjectLoader.getConnection(JdbcProjectLoader.java:1548)
	at azkaban.project.JdbcProjectLoader.fetchAllActiveProjects(JdbcProjectLoader.java:75)
	at azkaban.project.ProjectManager.loadAllProjects(ProjectManager.java:93)
	at azkaban.project.ProjectManager.<init>(ProjectManager.java:86)
	at azkaban.webapp.AzkabanWebServer.loadProjectManager(AzkabanWebServer.java:275)
	at azkaban.webapp.AzkabanWebServer.<init>(AzkabanWebServer.java:203)
	at azkaban.webapp.AzkabanWebServer.main(AzkabanWebServer.java:762)
	at azkaban.soloserver.AzkabanSingleServer.main(AzkabanSingleServer.java:53)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:144)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-193]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 27 more
2017-03-31 12:11:16 jdbc[4]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-193]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:184)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:126)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:99)
	at org.h2.Driver.connect(Driver.java:69)
	at org.apache.commons.dbcp.DriverConnectionFactory.createConnection(DriverConnectionFactory.java:38)
	at org.apache.commons.dbcp.PoolableConnectionFactory.makeObject(PoolableConnectionFactory.java:582)
	at org.apache.commons.dbcp.BasicDataSource.validateConnectionFactory(BasicDataSource.java:1556)
	at org.apache.commons.dbcp.BasicDataSource.createPoolableConnectionFactory(BasicDataSource.java:1545)
	at org.apache.commons.dbcp.BasicDataSource.createDataSource(BasicDataSource.java:1388)
	at org.apache.commons.dbcp.BasicDataSource.getConnection(BasicDataSource.java:1044)
	at org.apache.commons.dbutils.AbstractQueryRunner.prepareConnection(AbstractQueryRunner.java:175)
	at org.apache.commons.dbutils.QueryRunner.query(QueryRunner.java:286)
	at azkaban.executor.JdbcExecutorLoader.fetchExecutor(JdbcExecutorLoader.java:854)
	at azkaban.executor.ExecutorManager.setupExecutors(ExecutorManager.java:203)
	at azkaban.executor.ExecutorManager.<init>(ExecutorManager.java:126)
	at azkaban.webapp.AzkabanWebServer.loadExecutorManager(AzkabanWebServer.java:281)
	at azkaban.webapp.AzkabanWebServer.<init>(AzkabanWebServer.java:202)
	at azkaban.webapp.AzkabanWebServer.main(AzkabanWebServer.java:762)
	at azkaban.soloserver.AzkabanSingleServer.main(AzkabanSingleServer.java:53)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-193]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 22 more
2017-03-31 12:14:22 jdbc[4]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-193]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:184)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:126)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:99)
	at org.h2.Driver.connect(Driver.java:69)
	at org.apache.commons.dbcp.DriverConnectionFactory.createConnection(DriverConnectionFactory.java:38)
	at org.apache.commons.dbcp.PoolableConnectionFactory.makeObject(PoolableConnectionFactory.java:582)
	at org.apache.commons.dbcp.BasicDataSource.validateConnectionFactory(BasicDataSource.java:1556)
	at org.apache.commons.dbcp.BasicDataSource.createPoolableConnectionFactory(BasicDataSource.java:1545)
	at org.apache.commons.dbcp.BasicDataSource.createDataSource(BasicDataSource.java:1388)
	at org.apache.commons.dbcp.BasicDataSource.getConnection(BasicDataSource.java:1044)
	at org.apache.commons.dbutils.AbstractQueryRunner.prepareConnection(AbstractQueryRunner.java:175)
	at org.apache.commons.dbutils.QueryRunner.query(QueryRunner.java:286)
	at azkaban.executor.JdbcExecutorLoader.fetchExecutor(JdbcExecutorLoader.java:854)
	at azkaban.executor.ExecutorManager.setupExecutors(ExecutorManager.java:203)
	at azkaban.executor.ExecutorManager.<init>(ExecutorManager.java:126)
	at azkaban.webapp.AzkabanWebServer.loadExecutorManager(AzkabanWebServer.java:281)
	at azkaban.webapp.AzkabanWebServer.<init>(AzkabanWebServer.java:202)
	at azkaban.webapp.AzkabanWebServer.main(AzkabanWebServer.java:762)
	at azkaban.soloserver.AzkabanSingleServer.main(AzkabanSingleServer.java:53)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-193]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 22 more
2017-03-31 12:15:54 jdbc[4]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-193]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:184)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:126)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:99)
	at org.h2.Driver.connect(Driver.java:69)
	at org.apache.commons.dbcp.DriverConnectionFactory.createConnection(DriverConnectionFactory.java:38)
	at org.apache.commons.dbcp.PoolableConnectionFactory.makeObject(PoolableConnectionFactory.java:582)
	at org.apache.commons.dbcp.BasicDataSource.validateConnectionFactory(BasicDataSource.java:1556)
	at org.apache.commons.dbcp.BasicDataSource.createPoolableConnectionFactory(BasicDataSource.java:1545)
	at org.apache.commons.dbcp.BasicDataSource.createDataSource(BasicDataSource.java:1388)
	at org.apache.commons.dbcp.BasicDataSource.getConnection(BasicDataSource.java:1044)
	at org.apache.commons.dbutils.AbstractQueryRunner.prepareConnection(AbstractQueryRunner.java:175)
	at org.apache.commons.dbutils.QueryRunner.query(QueryRunner.java:286)
	at azkaban.executor.JdbcExecutorLoader.fetchExecutor(JdbcExecutorLoader.java:854)
	at azkaban.executor.ExecutorManager.setupExecutors(ExecutorManager.java:203)
	at azkaban.executor.ExecutorManager.<init>(ExecutorManager.java:126)
	at azkaban.webapp.AzkabanWebServer.loadExecutorManager(AzkabanWebServer.java:281)
	at azkaban.webapp.AzkabanWebServer.<init>(AzkabanWebServer.java:202)
	at azkaban.webapp.AzkabanWebServer.main(AzkabanWebServer.java:762)
	at azkaban.soloserver.AzkabanSingleServer.main(AzkabanSingleServer.java:53)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-193]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 22 more
2017-03-31 12:18:52 jdbc[4]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-193]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:184)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:126)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:99)
	at org.h2.Driver.connect(Driver.java:69)
	at org.apache.commons.dbcp.DriverConnectionFactory.createConnection(DriverConnectionFactory.java:38)
	at org.apache.commons.dbcp.PoolableConnectionFactory.makeObject(PoolableConnectionFactory.java:582)
	at org.apache.commons.dbcp.BasicDataSource.validateConnectionFactory(BasicDataSource.java:1556)
	at org.apache.commons.dbcp.BasicDataSource.createPoolableConnectionFactory(BasicDataSource.java:1545)
	at org.apache.commons.dbcp.BasicDataSource.createDataSource(BasicDataSource.java:1388)
	at org.apache.commons.dbcp.BasicDataSource.getConnection(BasicDataSource.java:1044)
	at org.apache.commons.dbutils.AbstractQueryRunner.prepareConnection(AbstractQueryRunner.java:175)
	at org.apache.commons.dbutils.QueryRunner.query(QueryRunner.java:286)
	at azkaban.executor.JdbcExecutorLoader.fetchExecutor(JdbcExecutorLoader.java:854)
	at azkaban.executor.ExecutorManager.setupExecutors(ExecutorManager.java:203)
	at azkaban.executor.ExecutorManager.<init>(ExecutorManager.java:126)
	at azkaban.webapp.AzkabanWebServer.loadExecutorManager(AzkabanWebServer.java:281)
	at azkaban.webapp.AzkabanWebServer.<init>(AzkabanWebServer.java:202)
	at azkaban.webapp.AzkabanWebServer.main(AzkabanWebServer.java:762)
	at azkaban.soloserver.AzkabanSingleServer.main(AzkabanSingleServer.java:53)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-193]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 22 more
2017-03-31 12:26:15 jdbc[6]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-193]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:184)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:126)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:99)
	at org.h2.Driver.connect(Driver.java:69)
	at org.apache.commons.dbcp.DriverConnectionFactory.createConnection(DriverConnectionFactory.java:38)
	at org.apache.commons.dbcp.PoolableConnectionFactory.makeObject(PoolableConnectionFactory.java:582)
	at org.apache.commons.dbcp.BasicDataSource.validateConnectionFactory(BasicDataSource.java:1556)
	at org.apache.commons.dbcp.BasicDataSource.createPoolableConnectionFactory(BasicDataSource.java:1545)
	at org.apache.commons.dbcp.BasicDataSource.createDataSource(BasicDataSource.java:1388)
	at org.apache.commons.dbcp.BasicDataSource.getConnection(BasicDataSource.java:1044)
	at azkaban.database.AbstractJdbcLoader.getDBConnection(AbstractJdbcLoader.java:69)
	at azkaban.project.JdbcProjectLoader.getConnection(JdbcProjectLoader.java:1548)
	at azkaban.project.JdbcProjectLoader.fetchAllActiveProjects(JdbcProjectLoader.java:75)
	at azkaban.project.ProjectManager.loadAllProjects(ProjectManager.java:93)
	at azkaban.project.ProjectManager.<init>(ProjectManager.java:86)
	at azkaban.webapp.AzkabanWebServer.loadProjectManager(AzkabanWebServer.java:275)
	at azkaban.webapp.AzkabanWebServer.<init>(AzkabanWebServer.java:203)
	at azkaban.webapp.AzkabanWebServer.main(AzkabanWebServer.java:762)
	at azkaban.soloserver.AzkabanSingleServer.main(AzkabanSingleServer.java:53)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-193]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 22 more
2017-03-31 12:27:14 jdbc[6]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-193]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:184)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:126)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:99)
	at org.h2.Driver.connect(Driver.java:69)
	at org.apache.commons.dbcp.DriverConnectionFactory.createConnection(DriverConnectionFactory.java:38)
	at org.apache.commons.dbcp.PoolableConnectionFactory.makeObject(PoolableConnectionFactory.java:582)
	at org.apache.commons.dbcp.BasicDataSource.validateConnectionFactory(BasicDataSource.java:1556)
	at org.apache.commons.dbcp.BasicDataSource.createPoolableConnectionFactory(BasicDataSource.java:1545)
	at org.apache.commons.dbcp.BasicDataSource.createDataSource(BasicDataSource.java:1388)
	at org.apache.commons.dbcp.BasicDataSource.getConnection(BasicDataSource.java:1044)
	at azkaban.database.AbstractJdbcLoader.getDBConnection(AbstractJdbcLoader.java:69)
	at azkaban.project.JdbcProjectLoader.getConnection(JdbcProjectLoader.java:1548)
	at azkaban.project.JdbcProjectLoader.fetchAllActiveProjects(JdbcProjectLoader.java:75)
	at azkaban.project.ProjectManager.loadAllProjects(ProjectManager.java:93)
	at azkaban.project.ProjectManager.<init>(ProjectManager.java:86)
	at azkaban.webapp.AzkabanWebServer.loadProjectManager(AzkabanWebServer.java:275)
	at azkaban.webapp.AzkabanWebServer.<init>(AzkabanWebServer.java:203)
	at azkaban.webapp.AzkabanWebServer.main(AzkabanWebServer.java:762)
	at azkaban.soloserver.AzkabanSingleServer.main(AzkabanSingleServer.java:53)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-193]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 22 more
2017-03-31 12:30:41 jdbc[6]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-193]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:184)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:126)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:99)
	at org.h2.Driver.connect(Driver.java:69)
	at org.apache.commons.dbcp.DriverConnectionFactory.createConnection(DriverConnectionFactory.java:38)
	at org.apache.commons.dbcp.PoolableConnectionFactory.makeObject(PoolableConnectionFactory.java:582)
	at org.apache.commons.dbcp.BasicDataSource.validateConnectionFactory(BasicDataSource.java:1556)
	at org.apache.commons.dbcp.BasicDataSource.createPoolableConnectionFactory(BasicDataSource.java:1545)
	at org.apache.commons.dbcp.BasicDataSource.createDataSource(BasicDataSource.java:1388)
	at org.apache.commons.dbcp.BasicDataSource.getConnection(BasicDataSource.java:1044)
	at azkaban.database.AbstractJdbcLoader.getDBConnection(AbstractJdbcLoader.java:69)
	at azkaban.project.JdbcProjectLoader.getConnection(JdbcProjectLoader.java:1548)
	at azkaban.project.JdbcProjectLoader.fetchAllActiveProjects(JdbcProjectLoader.java:75)
	at azkaban.project.ProjectManager.loadAllProjects(ProjectManager.java:93)
	at azkaban.project.ProjectManager.<init>(ProjectManager.java:86)
	at azkaban.webapp.AzkabanWebServer.loadProjectManager(AzkabanWebServer.java:275)
	at azkaban.webapp.AzkabanWebServer.<init>(AzkabanWebServer.java:203)
	at azkaban.webapp.AzkabanWebServer.main(AzkabanWebServer.java:762)
	at azkaban.soloserver.AzkabanSingleServer.main(AzkabanSingleServer.java:53)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-193]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 22 more
2017-03-31 12:33:17 jdbc[6]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-193]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:184)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:126)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:99)
	at org.h2.Driver.connect(Driver.java:69)
	at org.apache.commons.dbcp.DriverConnectionFactory.createConnection(DriverConnectionFactory.java:38)
	at org.apache.commons.dbcp.PoolableConnectionFactory.makeObject(PoolableConnectionFactory.java:582)
	at org.apache.commons.dbcp.BasicDataSource.validateConnectionFactory(BasicDataSource.java:1556)
	at org.apache.commons.dbcp.BasicDataSource.createPoolableConnectionFactory(BasicDataSource.java:1545)
	at org.apache.commons.dbcp.BasicDataSource.createDataSource(BasicDataSource.java:1388)
	at org.apache.commons.dbcp.BasicDataSource.getConnection(BasicDataSource.java:1044)
	at azkaban.database.AbstractJdbcLoader.getDBConnection(AbstractJdbcLoader.java:69)
	at azkaban.project.JdbcProjectLoader.getConnection(JdbcProjectLoader.java:1548)
	at azkaban.project.JdbcProjectLoader.fetchAllActiveProjects(JdbcProjectLoader.java:75)
	at azkaban.project.ProjectManager.loadAllProjects(ProjectManager.java:93)
	at azkaban.project.ProjectManager.<init>(ProjectManager.java:86)
	at azkaban.webapp.AzkabanWebServer.loadProjectManager(AzkabanWebServer.java:275)
	at azkaban.webapp.AzkabanWebServer.<init>(AzkabanWebServer.java:203)
	at azkaban.webapp.AzkabanWebServer.main(AzkabanWebServer.java:762)
	at azkaban.soloserver.AzkabanSingleServer.main(AzkabanSingleServer.java:53)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-193]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 22 more
2017-03-31 12:33:53 jdbc[6]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-193]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:184)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:126)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:99)
	at org.h2.Driver.connect(Driver.java:69)
	at org.apache.commons.dbcp.DriverConnectionFactory.createConnection(DriverConnectionFactory.java:38)
	at org.apache.commons.dbcp.PoolableConnectionFactory.makeObject(PoolableConnectionFactory.java:582)
	at org.apache.commons.dbcp.BasicDataSource.validateConnectionFactory(BasicDataSource.java:1556)
	at org.apache.commons.dbcp.BasicDataSource.createPoolableConnectionFactory(BasicDataSource.java:1545)
	at org.apache.commons.dbcp.BasicDataSource.createDataSource(BasicDataSource.java:1388)
	at org.apache.commons.dbcp.BasicDataSource.getConnection(BasicDataSource.java:1044)
	at azkaban.database.AbstractJdbcLoader.getDBConnection(AbstractJdbcLoader.java:69)
	at azkaban.project.JdbcProjectLoader.getConnection(JdbcProjectLoader.java:1548)
	at azkaban.project.JdbcProjectLoader.fetchAllActiveProjects(JdbcProjectLoader.java:75)
	at azkaban.project.ProjectManager.loadAllProjects(ProjectManager.java:93)
	at azkaban.project.ProjectManager.<init>(ProjectManager.java:86)
	at azkaban.webapp.AzkabanWebServer.loadProjectManager(AzkabanWebServer.java:275)
	at azkaban.webapp.AzkabanWebServer.<init>(AzkabanWebServer.java:203)
	at azkaban.webapp.AzkabanWebServer.main(AzkabanWebServer.java:762)
	at azkaban.soloserver.AzkabanSingleServer.main(AzkabanSingleServer.java:53)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-193]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 22 more
2017-03-31 17:38:59 jdbc[6]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-193]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:184)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:126)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:99)
	at org.h2.Driver.connect(Driver.java:69)
	at org.apache.commons.dbcp.DriverConnectionFactory.createConnection(DriverConnectionFactory.java:38)
	at org.apache.commons.dbcp.PoolableConnectionFactory.makeObject(PoolableConnectionFactory.java:582)
	at org.apache.commons.dbcp.BasicDataSource.validateConnectionFactory(BasicDataSource.java:1556)
	at org.apache.commons.dbcp.BasicDataSource.createPoolableConnectionFactory(BasicDataSource.java:1545)
	at org.apache.commons.dbcp.BasicDataSource.createDataSource(BasicDataSource.java:1388)
	at org.apache.commons.dbcp.BasicDataSource.getConnection(BasicDataSource.java:1044)
	at azkaban.database.AbstractJdbcLoader.getDBConnection(AbstractJdbcLoader.java:69)
	at azkaban.project.JdbcProjectLoader.getConnection(JdbcProjectLoader.java:1548)
	at azkaban.project.JdbcProjectLoader.fetchAllActiveProjects(JdbcProjectLoader.java:75)
	at azkaban.project.ProjectManager.loadAllProjects(ProjectManager.java:93)
	at azkaban.project.ProjectManager.<init>(ProjectManager.java:86)
	at azkaban.webapp.AzkabanWebServer.loadProjectManager(AzkabanWebServer.java:275)
	at azkaban.webapp.AzkabanWebServer.<init>(AzkabanWebServer.java:203)
	at azkaban.webapp.AzkabanWebServer.main(AzkabanWebServer.java:762)
	at azkaban.soloserver.AzkabanSingleServer.main(AzkabanSingleServer.java:53)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-193]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 22 more
2017-03-31 17:42:19 jdbc[6]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-193]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:184)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:126)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:99)
	at org.h2.Driver.connect(Driver.java:69)
	at org.apache.commons.dbcp.DriverConnectionFactory.createConnection(DriverConnectionFactory.java:38)
	at org.apache.commons.dbcp.PoolableConnectionFactory.makeObject(PoolableConnectionFactory.java:582)
	at org.apache.commons.dbcp.BasicDataSource.validateConnectionFactory(BasicDataSource.java:1556)
	at org.apache.commons.dbcp.BasicDataSource.createPoolableConnectionFactory(BasicDataSource.java:1545)
	at org.apache.commons.dbcp.BasicDataSource.createDataSource(BasicDataSource.java:1388)
	at org.apache.commons.dbcp.BasicDataSource.getConnection(BasicDataSource.java:1044)
	at azkaban.database.AbstractJdbcLoader.getDBConnection(AbstractJdbcLoader.java:69)
	at azkaban.project.JdbcProjectLoader.getConnection(JdbcProjectLoader.java:1548)
	at azkaban.project.JdbcProjectLoader.fetchAllActiveProjects(JdbcProjectLoader.java:75)
	at azkaban.project.ProjectManager.loadAllProjects(ProjectManager.java:93)
	at azkaban.project.ProjectManager.<init>(ProjectManager.java:86)
	at azkaban.webapp.AzkabanWebServer.loadProjectManager(AzkabanWebServer.java:275)
	at azkaban.webapp.AzkabanWebServer.<init>(AzkabanWebServer.java:203)
	at azkaban.webapp.AzkabanWebServer.main(AzkabanWebServer.java:762)
	at azkaban.soloserver.AzkabanSingleServer.main(AzkabanSingleServer.java:53)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-193]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 22 more
2017-03-31 17:46:40 jdbc[6]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-193]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:184)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:126)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:99)
	at org.h2.Driver.connect(Driver.java:69)
	at org.apache.commons.dbcp.DriverConnectionFactory.createConnection(DriverConnectionFactory.java:38)
	at org.apache.commons.dbcp.PoolableConnectionFactory.makeObject(PoolableConnectionFactory.java:582)
	at org.apache.commons.dbcp.BasicDataSource.validateConnectionFactory(BasicDataSource.java:1556)
	at org.apache.commons.dbcp.BasicDataSource.createPoolableConnectionFactory(BasicDataSource.java:1545)
	at org.apache.commons.dbcp.BasicDataSource.createDataSource(BasicDataSource.java:1388)
	at org.apache.commons.dbcp.BasicDataSource.getConnection(BasicDataSource.java:1044)
	at azkaban.database.AbstractJdbcLoader.getDBConnection(AbstractJdbcLoader.java:69)
	at azkaban.project.JdbcProjectLoader.getConnection(JdbcProjectLoader.java:1548)
	at azkaban.project.JdbcProjectLoader.fetchAllActiveProjects(JdbcProjectLoader.java:75)
	at azkaban.project.ProjectManager.loadAllProjects(ProjectManager.java:93)
	at azkaban.project.ProjectManager.<init>(ProjectManager.java:86)
	at azkaban.webapp.AzkabanWebServer.loadProjectManager(AzkabanWebServer.java:275)
	at azkaban.webapp.AzkabanWebServer.<init>(AzkabanWebServer.java:203)
	at azkaban.webapp.AzkabanWebServer.main(AzkabanWebServer.java:762)
	at azkaban.soloserver.AzkabanSingleServer.main(AzkabanSingleServer.java:53)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-193]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 22 more
